api/v1/generate.sh

```diff
@@ -2,28 +2,45 @@
 
 set -e
 
-ROOT=${GOPATH}/src
-SOLO_KIT=${ROOT}/github.com/solo-io/solo-kit
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+ROOT=$(dirname "${BASH_SOURCE[0]}")/../../..
+SOLO_KIT=${ROOT}/solo-kit
 IN=${SOLO_KIT}/api/v1/
 EXTERNAL=${SOLO_KIT}/api/external/
-OUT=${SOLO_KIT}/pkg/api/external/istio/encryption/v1/
+
+# code-generator does work with go.mod but makes assumptions about
+# the project living in $GOPATH/src. To work around this and support
```

Comment:
what kinds of assumptions? how are these assumptions leveraged? at some point we'd like to undo some of these assumptions -- or at least properly document them